tools/travis/.travis.yml

```diff
@@ -12,12 +12,13 @@ php:
 env:
   global:
     - PATH="$HOME/.composer/vendor/bin:$PATH"
+    - SYMFONY_DEPRECATIONS_HELPER=weak
 
 matrix:
   fast_finish: true
   include:
     - php: 5.3
-      env: COMPOSER_FLAGS="--prefer-lowest"
+      env: COMPOSER_FLAGS="--prefer-lowest --prefer-stable"
```

Comment:
I think we should avoid `minimum-stability` at all costs and specify dev dependencies explicitely instead. I have come to believe Composer is less efficient with it.